rpc_multi_node.yml

```diff
@@ -5,7 +5,7 @@ parameters:
     type: string
     label: Image name or ID
     description: Image to be used for compute instance
-    default: a3ba4cf5-70b9-4805-afa2-30d1ab81a625
+    default: a1558fdc-3182-4a0f-b48a-aa900a5826c3
```

Comment:
Yeah, if you use the name instead of the UUID the. You should receive updated images from time to time.